pom.xml

```diff
@@ -48,7 +48,7 @@
     <releaseProfiles> </releaseProfiles>
     <arguments> </arguments>
     <argLine>-Xmx768M -Djava.awt.headless=true</argLine>
-    <jenkins.version>1.625.3</jenkins.version>
+    <jenkins.version>2.7</jenkins.version>
```

Comment:
strictly speaking was 2.0 but go to the next nearest thing that was an LTS.

Reply:
Recommend 2.7.4.

Reply:
@jglick Just out of principle, or are there actual fixes?

For users, 2.7 is probably better due to how the tiered update centers work.